README.md

```diff
@@ -128,6 +128,35 @@ The following VCF fields can be accessed in the filter expression:
     'not {"pathogenic", "likely_pathogenic", "drug_response"}.isdisjoint(ANN["CLIN_SIG"])' \
     variants.vcf
   ```
+* Filter on sample specific values:
+  * by sample name:
+    ```sh
+    vembrane filter 'FORMAT["AD"]["specific_sample_name"] > 0' variants.vcf
```

Comment:
`FORMAT["AD"]` will have one entry per allele, right? So we should probably use a simpler entry, like the `FORMAT["DP"]`used below. Otherwise, we would need to somehow make the handling of `A` type entries clear in this context, but it is probably best explained on its own.

Reply:
Yup, thanks ;)